server/src/pages/leave-screenshots/server.js

```diff
@@ -11,7 +11,7 @@ exports.app = app;
 
 app.get("/", csrfProtection, function(req, res) {
   if (!req.deviceId) {
-    res.status(403).send("You must have the addon installed to delete your account");
+    res.status(403).send(req.getText("leavePageErrorAddonRequired"));
```

Comment:
I'm somewhat indifferent to translating these responses. They are plain text and shouldn't actually show up to anyone unless they do something weird.  But whatever.